### DISCUSSION TITLE
accesso a contatore e-Distribuzione

### DISCUSSION THREAD

**Discussion by rcecinati** [indomus/forum#2110] [HUB: Home Assistant]
buona sera,
sono nuovo in questo Forum. Sto costruendo un ambiente domotico in casa mia , con l'obiettivo di controllare consumi, produzione fotovoltaica, clima, sicurezza. Le componenti che sto assemblando sono HomeAssistant su Raspberry PI, Xiaomi Home Mijia su Zigbee e Wi-Fi, MQTT. Un po' alla volta, grazie anche agli ottimi articoli su Indomus, mi sto facendo una cultura (che non ho) sulla materia della domotica.
La domanda è la seguente:
esistono esperienze di integrazione in HA dei contatori elettrici di seconda generazione installati da E-Distribuzione? Sono fonti di dati preziosi e potenti, e in un impianto FV ne abbiamo due, uno  misurante l'energia in entrata dalla grid ed uno per l'energia immessa. I contatori, almeno quelli che conosco io, sono prodotti nalla Gridspertise, e si chiamano GEMIS. Il protocollo  si chiama Chain2, e so dell'esistenza di smart meter che parlano via powerline con tali contatori, ad esempio i Chain2Gate della MAC  . Se ne ho la possibilità, posso postare qui i riferimenti ai siti web che descrivono prodotti e protocollo. 
L'obiettivo dovrebbe essere chiaro, accedere a uno scrigno di informazioni che abbiamo in casa.
Grazie dell'attenzione.

**Comment by TeoLinuX** [comment]
ho letto anche io che si possono usare dispositivi che usano il chain2. Purtroppo io ho il contatore fuori dall'appartamento, abbastanza distante e in condominio.

e-distribuzione non ha integrazioni per leggerli (pur con un ritardo di 24 ore). Enel Italia (venditore, non il distributore locale) ha una integrazione HACS

La mia unica via è una pinza amperometrica nel quadro elettrico in casa (shelly o altro)

**Comment by indomus** [comment]
scusate, [questo](https://indomus.it/componenti/onics-frient-electricity-meter-interface-2-led/) lo avete considerato?
